### PR TITLE
Updated contract upgrade flow for GovernorUpgrade

### PR DESCRIPTION
**Pull request by shivamgarglab49**

### CODE REVIEW

File: integrationTest/governance/governorUpgrade.ts [hunk 1]
```diff
@@ -1,70 +1,143 @@
...
+  const ftHolderContractId = await ftTokenHolderFactory.getTokenHolder(
+    FT_TOKEN_ID.toSolidityAddress()
+  );
+  const godHolder = new GodHolder(ftHolderContractId);
 
   const governor = new ContractUpgradeGovernor();
-  const godHolderFactory = new FTTokenHolderFactory();
-  const godHolderContractId = await godHolderFactory.getTokenHolder(
-    GOD_TOKEN_ID.toSolidityAddress()
+  await governor.cancelProposal(
```
Comment (amandeeprehal-lab49) [inline integrationTest/governance/governorUpgrade.ts:33]: Looks not required

Comment (shivamgarglab49) [reply integrationTest/governance/governorUpgrade.ts:33]: done

File: contracts/governance/GovernorUpgrade.sol [hunk 2]
```diff
@@ -3,48 +3,70 @@ pragma solidity ^0.8.18;
...
+        address _proxyAdmin
+    ) private {
+        (bool success, ) = _proxy.call(
+            abi.encodeWithSignature("upgradeTo(address)", _proxyLogic)
         );
-        TokenUpgradeData memory tokenUpgradeData = _proposalData[proposalId];
-        return (
-            tokenUpgradeData.proxyContract,
-            tokenUpgradeData.contractToUpgrade
+        require(success, "GU: failed to upgrade proxy");
```
Comment (amandeeprehal-lab49) [inline contracts/governance/GovernorUpgrade.sol:66]: Pls change "GU: failed to upgrade proxy" to "GU: failed to upgrade proxy, verify governor is owner"

Comment (shivamgarglab49) [reply contracts/governance/GovernorUpgrade.sol]: done

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Pull request merged**